client/src/modules/notification/components/NotificationWithTimer.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import Notification from './Notification';
+
+class NotificationWithTimer extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    const { time } = this.props;
+
+    this.state = {
+      time
+    };
+  }
+
+  componentDidMount() {
+    this.createInterval();
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.timer);
```

Comment:
I think this.timer should be declared in the constructor with some default value